sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_healthcare.py

```diff
@@ -26,7 +26,7 @@
 # pre-apply the client_cls positional argument so it needn't be explicitly passed below
 TextAnalyticsClientPreparer = functools.partial(_TextAnalyticsClientPreparer, TextAnalyticsClient)
 
-
+@pytest.mark.skip("404 Not Found")
```

Comment:
healthcare endpoint doesn't seem deployed yet?

Reply:
yeah that seems to be the case, ok with skipping for now